build.sh

```diff
@@ -525,6 +543,16 @@ if [ ${BUILD_BE} -eq 1 ]; then
     rm -f ${STARROCKS_OUTPUT}/be/lib/hadoop/common/lib/log4j-1.2.17.jar
     rm -f ${STARROCKS_OUTPUT}/be/lib/hadoop/hdfs/lib/log4j-1.2.17.jar
 
+    if [ "${WITH_CACHELIB}" == "ON"  ]; then
+        mkdir -p ${STARROCKS_OUTPUT}/be/lib/cachelib
+        cp -r -p ${CACHELIB_DIR}/deps/lib64 ${STARROCKS_OUTPUT}/be/lib/cachelib/
+    fi
+
+    if [ "${USE_AVX2KI}" == "ON"  ]; then
+        mkdir -p ${STARROCKS_OUTPUT}/be/lib/avx2ki
+        cp ${STARROCKS_THIRDPARTY}/installed/lib/libavx2neon.so ${STARROCKS_OUTPUT}/be/lib/avx2ki/libavx2neon.so.2.0.0
```

Comment:
whether need to make a symbolic link of libavx2neon.so to libavx2neon.so.2.0.0?

Reply:
If I don't do this, the final output execution file does not link to the .so file. I havn't figured out why but this works.